src/main/fc/fc_core.c

```diff
@@ -205,6 +205,13 @@ void mwArm(void)
             return;
         }
         if (!ARMING_FLAG(PREVENT_ARMING)) {
+            #ifdef USE_DSHOT
+            if (mixerConfig()->reverse_motors == false)
+                pwmWriteDshotCommandAll(getMotorCount(), DSHOT_CMD_ROTATE_NORMAL);
```

Comment:
Why are we doing this? If users want motors to be permanently reversed, they should just use DShot commands to permanently set the motor direction to be reversed.